poetry-installer-error-kpwo7wsr.log

```diff
@@ -0,0 +1,14 @@
+Looking in indexes: https://anu9rng:****@rb-artifactory.bosch.com/artifactory/api/pypi/python-virtual/simple


```

Comment:
Minor, but this sort of error log isn't worth tracking in source control - it doesn't mean anything to future developers, but could easily be a file that changes for several people and causes clashes that we have to solve (even though they don't matter), so it's worth us ignoring this

We can do that by:
* Adding `*.log` to your `.gitignore` file, so that future changes to any file ending with the `.log` extension get ignored
* Removing the existing knowledge Git has for this file, with `git rm --cached poetry-installer-error-kpwo7wsr.log`
* Adding/committing/pushing those changes